test/src/code_field/code_controller_comment_uncomment_test.dart

```diff
@@ -15,105 +15,30 @@ void main() {
           'and uncomment otherwise '
           'AND select whole lines', () {
         final examples = [
+          //
```

Comment:
Add tests for invalid selection of `-1` (must do nothing).

Also looks like we now have only tests that change each line. We need at least some tests with deselected lines that do not change, although we do not need all edge cases for that.